app/grandchallenge/components/admin.py

```diff
@@ -93,12 +94,24 @@ def requeue_jobs(modeladmin, request, queryset):
 
     for job in queryset:
         job.status = ComponentJob.RETRY
+        job.started_at = now()
```

Comment:
Isn't the start time set in `update_status`? Then I think `started_at` should be `None` as well.

Reply:
Done